core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -47,6 +47,13 @@
     </ul>
   </li>
 
+  <li ng-if="userIsLoggedIn && enableTranslationTab" id="tutorialTranslationTab" ng-class="{'active': getTabStatuses().active === 'translation'}">
+    <a href="#" uib-tooltip="Translation" tooltip-placement="bottom" ng-click="selectTranslationTab()"
+       class="oppia-editor-navbar-tab-anchor protractor-test-audio-tab">
```

Comment:
`protractor-test-audio-tab` is not used anywhere.

Reply:
This is for future use, as currently, I cannot add e2e tests as the translation tab is disabled.